lighthouse-treemap/app/images/lighthouse-logo.svg

```diff
@@ -0,0 +1,116 @@
+<svg width="128" height="128" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I can't comment directly on them, but those favicon file renames seem like a mistake. I don't see a manifest.json for the treemap app that references them and I don't see the viewer references removed either.

Reply:
oops... I copied the images folder, and accidently deleted excess images from the wrong folder. hence "rename"